cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/ThrowStatement.kt

```diff
@@ -33,8 +33,8 @@ import java.util.Objects
 import org.apache.commons.lang3.builder.ToStringBuilder
 import org.neo4j.ogm.annotation.Relationship
 
-/** Represents a `throw` or `raise` statement. */
-class ThrowStatement : Statement(), ArgumentHolder {
+/** Represents a `throw` or `raise` statement/expression. */
+class ThrowStatement : Expression(), ArgumentHolder {
```

Comment:
Of course C++ has to be the party pooper :( Ok it seems like it make sense to make `throw` an expression (of void type?), but then I guess we should rename it to a `ThrowExpression` otherwise its very confusing.

Reply:
yes, wanted to do this separately/at the end of my PR session to avoid a bunch of collisions ;)

Reply:
Ok good idea. Then I'll accept this as-is under the strong promise of the follow up PR ;)